.github/workflows/release.yml

```diff
@@ -200,8 +200,29 @@ jobs:
       base64-subjects: "${{ needs.sbom-assests.outputs.hashes }}"
       provenance-name: "karmada-sbom.intoto.jsonl"
       upload-assets: true
+
+  get-stable-tag:
+    name: Get Stable tag
+    runs-on: ubuntu-22.04
+    env:
+      GH_TOKEN: ${{ github.token }}
+    outputs:
+      stableTag: ${{ steps.get.outputs.stableTag }}
+    steps:
+    - name: get
+      id: get
+      run: |  
+        # get the max version from tags list and filter versions of alpha,beta,preview,rc
+        export STABLE_TAG=`gh api repos/karmada-io/karmada/tags --paginate | jq -r '.[].name' | grep -Ev "alpha|beta|preview|rc" | sort -Vr | head -n 1`
+        echo "Got the stabe tag:$STABLE_TAG"
+        echo "event.tag:"${{ github.event.release.tag_name }}
+        echo "stableTag=$STABLE_TAG" >> "$GITHUB_OUTPUT"
```

Comment:
Another question, can we put this logic to the `update-krew-index` job? Does it have to be separated into two jobs?

Reply:
You're right, and you need to make sure the latest release is right, for example, according to the following release order:

- 1.10.3 08/27  make it to latest release
- 1.10.4  08/28
- 1.9.6   08/29

and then  latest API just got the `1.10.3`, this might be somewhat risky, but I agree it would be more straightforward this way, and I'll update it to use the latest API.

Reply:
> You're right, and you need to make sure the latest release is right,

Yeah, that's the release team's responsibility to mark the latest release correctly.

Reply:
> Another question, can we put this logic to the update-krew-index job? 

update-krew-index doesn't care whether it is the latest version, so it is impossible to merge the logic into the update-krew-index action job here.
And i open a issue at upstream https://github.com/rajatjindal/krew-release-bot/issues/75

Reply:
In your example above, the release team will mark `v1.10.4` as the latest release.